.github/workflows/check-version.yml

```
@@ -0,0 +1,87 @@
+name: Check version
+
+on:
+  push:
+    branches:
+      - develop
+      - release/*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  VERSION_FILE: ./package.json
+
+jobs:
+  check-version:
+    name: Check version
```

It does more than checking the version though:

```suggestion
    name: Bump version
```

---

Changed in https://github.com/nimblehq/infrastructure-templates/pull/237/commits/3b0ed365bfb509d6ebd0f30f4d99a58704f62951